includes/class-sensei-learner-profiles.php

```diff
@@ -36,7 +36,32 @@ public function __construct() {
 
 		// Add class to body tag
 		add_filter( 'body_class', array( $this, 'learner_profile_body_class' ), 10, 1 );
-	} // End __construct()
+
+		// Scripts for frontend.
+		add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_scripts' ) );
+	} // End __construct().
+
+	/**
+	 * Enqueue frontend JavaScripts.
+	 *
+	 * @since  3.0.0
+	 * @access private
+	 */
+	public function enqueue_scripts() {
+		if ( ! Sensei_Utils::get_setting_as_flag( 'js_disable', 'sensei_settings_js_disable' ) ) {
+			global $wp;
+			global $post;
+			if ( preg_match( '/\/learner\//', add_query_arg( $wp->query_vars, home_url( $wp->request ) ) ) ||
```

Comment:
If this PHP file runs, it's safe to assume it's because the user is viewing the learner profile page. So we actually don't need to add this fairly complicated logic check here. We can just go ahead and register the script.

Reply:
Not a silly question, especially since I gave you faulty advice. 🤦‍♀

[I conditionally loaded some CSS on the learner profile page](https://github.com/woocommerce/sensei-certificates/pull/205/commits/dd9415f9606932306649f4df4fd9261a55189699#diff-70314252a9445b55ea6515bc709793e0R234) the other day. Try `isset( $wp_query->query_vars['learner_profile']` and see if that works.